### DISCUSSION TITLE
Bias estimation

### DISCUSSION THREAD

**Discussion by k9891** [groops-devs/groops#185] [Q&A]
Hello, I have some peoblems about bias estimation using BDS.  I used the estimated OSB for PPP-AR in the user end and found that its EWL ambiguity fixed rate is relatively low (around 50%). However, using the same strategy to estimate Galileo's OSB  does not have a problem. I would like to ask what may be the reason for this？ My first thought is the APC correction for the third frequency, as described in the paper (see picture). Is this method suitable for BDS multi frequency processing? 

Thank you！ 
<img width="530" alt="微信截图_20230510211843" src="https://github.com/groops-devs/groops/assets/58100188/601280b9-bc40-4554-98a7-bbdcca1dde49">

**Comment by zhedumi** [comment]
Hi,
I probably need some info so I can help. Did you estimate the code/phase biases for BDS using groops and then used another program with that data for EWL fixing tests? But yeah other people have mentioned that within BDS the ambiguity fixing is not yet working as intended. I do not think the APC would bring this about. Currently I am updating the BDS attitudes for the main branch so maybe in the next small github update that could fix it.

best regards,
Patrick

**Comment by k9891** [comment]
Hello,
Thank you for your answers. As you mentioned, I used Groops for OSB estimation and PPP-AR on another program. But I have a question. I have made modifications to the program and now the BDS B1I/B3I WL/NL ambiguity fixed rate is still acceptable. However, when any third frequency forms EWL ambiguity, the results are very poor.  Is this caused by a fixed ambiguity error？
I am very looking forward to the next groops update.

Best regards,
K9891

**Comment by zhedumi** [comment]
Hi,
I checked through groops and hte attitude modes for BeiDou are already within the main bracnhes so its still more surprising. It could be the ambiguity fixing. Its really interesting that when a third frequency is used for EWL combination that the results are that bad. As soon as I checked some more I will update you if I find something regarding this problem. 

best regards
Patrick
